src/main/java/CodeCheck/OneFunction.java

```diff
@@ -17,23 +17,30 @@ protected OneFunction() {
         content = new ArrayList<>();
     }
 
-    public int equals(OneFunction obj) {
+    private enum Compare {
+        Default,
+        Identical,
+        Similar,
+        CloseThereby,
+    }
+
+    public Compare equals(OneFunction obj) {
         if (obj.content.size() > 2 && obj.name.equals(name) && obj.removeExtraInformationAndGetString().equals(removeExtraInformationAndGetString()))
-            return 1;
+            return Compare.Identical;
         if (obj.name.equals(name) && obj.content.size() > 4 && content.size() > 4)
-            return 2;
+            return Compare.Similar;
         if (obj.name.equals(name))
-            return 3;
+            return Compare.CloseThereby;
 
-        return 10;
+        return Compare.Default;
```

Comment:
Default -> no match

Reply:
Oh, agree! Missed it. Will be in next commit!

Reply:
Tried to get something like that working and also throw an error message or warning message for it as well. But didn't work as I wanted.